src/input/label.component.ts

```diff
@@ -98,7 +98,7 @@ import { TextInput } from "./input.directive";
 				[attr.data-invalid]="(invalid ? true : null)"
 				#wrapper>
 				<svg
-					*ngIf="!warn && invalid"
+					*ngIf="invalid"
```

Comment:
Are you able to apply this change to all of other form components? This is so that the behavior is consistent for all form components.

You can search for `"!warn && invalid"` and you'll find numerous instances of this.

![image](https://github.com/carbon-design-system/carbon-components-angular/assets/38994122/434d91fc-435a-4cd8-81e3-ff040cf02d68)

Reply:
sure thing, I'll do it as soon I am free.

Reply:
Great, thanks! 🎉

Reply:
done :)